projects/lib/src/lib/services/luigi-nodes/nodes-processing.service.ts

```diff
@@ -48,7 +50,10 @@ export class NodesProcessingService {
     ];
 
     globalNodes.forEach((node) => {
-      node.context = { ...node.context };
+      if (node.context) {
```

Comment:
not needed the line here assumed the context might not be existing that's why the line

node.context = { ...node.context };